### ISSUE TITLE
Automate chocolatey releases

### ISSUE THREAD

**Issue by rjatkins** [jasongin/nvs#156]
Releases 1.5.3 and 1.5.4 haven't been released to chocolatey yet, and need to be manually done. It would be better if these releases were automatically done as part of releasing the main package, rather than a manual step afterwards. The automation should handle updating the version in the nvs.nuspec and chocolateyinstall.ps1 (for the download URL).

**Comment by pchmielinski** [comment]
@molant I see you implemented chocolatey packaging for this repository in https://github.com/jasongin/nvs/pull/91 and later updated it to 1.5.2 in https://github.com/jasongin/nvs/pull/121. It would be nice if we could get newer versions deployed to chocolatey as well.

In the original PR, a script to automate this was already discussed and wasn't deemed that necessary due to low ROI. However, I guess the assumption was that updating the chocolatey files would be done manually by @jasongin on new release, which doesn't seem to be the case :). One possible reason is that the chocolatey/README.md doesn't have a short TL;DR section on how to upgrade the package on chocolatey.org - @jasongin did mention in the original PR he's not very familiar with Chocolatey.

I see the version bump PR is really straightforward, but I assume updating the package on chocolatey.org requires being the package maintainer, so it's not something the community can solve on its own.

**Comment by molant** [comment]
Hi all,

Unfortunately I don't have any spare time these days (family has grown a bit) and I don't have write permissions in this repo neither.
If @jasongin is OK I could give admin permissions to someone in the chocolatey package so it can be updated more often. I took a quick look at the docs in the chocolatey and it seems like there are tools to automate now the publishing of new versions? Might be worth exploring.

Sorry I can't bring better news 😞

**Comment by zsoltlengyelit** [comment]
hi,
@molant I'm open to take the task if @jasongin is ok with it too.

**Comment by molant** [comment]
I'll be fine too if everyone is onboard 😊

**Comment by jasongin** [comment]
Hi, I don't have much time to work on automating chocolatey releases. If someone else wants to work on that, great. However be aware that the Windows MSI build process runs on an internal build agent so that the MSI packages can get a Microsoft digital signature. (The MSI build files are all in the repo though.) That means it wouldn't be possible for someone else to build a release package. But maybe publishing to chocolatey can be done as a separate step that pulls from the GitHub release.

Currently I don't think I can publish an update to the chocolatey package anyway, since I'm not a maintainer there. Maybe @molant can add more maintainers?

**Comment by molant** [comment]
> Maybe @molant can add more maintainers?

Definitely, just let me know your chocolatey usernames and I can add you.

> But maybe publishing to chocolatey can be done as a separate step that pulls from the GitHub release.

IIRC that's the way it works. The files point to the MSI in GitHub with the right checksum.

**Comment by MrJithil** [comment]
I can volunteer for this activity.
